test/WebSites/Basic/Program.cs

```diff
@@ -1,20 +1,42 @@
 ﻿using Microsoft.AspNetCore.Hosting;
+
+#if NETCOREAPP3_0
+using Microsoft.Extensions.Hosting;
+#else
 using System.IO;
+#endif
 
 namespace Basic
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            var host = new WebHostBuilder()
+            CreateHostBuilderAndRun(args);
+        }
+
+        public static void CreateHostBuilderAndRun(string[] args) =>
+#if NETCOREAPP3_0
+            NC3_CreateHostBuilder(args).Build().Run();
+#else
+            NC2_CreateHostBuilder().Build().Run();
+
+#endif
```

Comment:
This is exactly the kind of complexity I was hoping to avoid/discourage by not multi-targeting. Is this really neccessary with this PR. I want to minimize the conditional code throughout the entire repo.

Reply:
I can totally rename it.
It's just a copy paste of both 2.x and 3.0 default `Program.cs`

The idea is that now i'm sure all the test you wrote runs sucessfully on both stack.
There was only 1 project for `netcreapp3.0`, so it add lots of consistency and insurrance in fact since full test runs on all sample.

On top of that it might help people understanding how/why changes can be done from 2.x to 3.0

Reply:
Do you want me to rebase that change to something like this ? 
https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/b251ef18a9f855a2532f93e7d3d723f63dad0df9/test/WebSites/CliExample/Program.cs
it's probably more accurate to default template